### PR TITLE
Override translate method to handle component path

### PR DESCRIPTION
**Pull request by florentferry**
Implements #24 
  
Related to #18 changes.

### CODE REVIEW

File: lib/komponent/komponent_helper.rb [hunk 1]
```diff
@@ -73,4 +72,22 @@ def render_partial(partial_name, locals = {}, &block)
 
     rendered_partial
   end
+
+  def translate(key, options = {})
```
Comment (sigmike) [inline lib/komponent/komponent_helper.rb]: Defining it here will override the `translate` method in all views. I think it should instead be put inside a module and prepended on the component context.

File: lib/komponent/komponent_helper.rb [hunk 2]
```diff
@@ -73,4 +72,22 @@ def render_partial(partial_name, locals = {}, &block)
...
+    virtual_path = @virtual_path
+
+    is_component = key.to_s.first == "." and
+      virtual_path =~ /$components/
```
Comment (sigmike) [inline lib/komponent/komponent_helper.rb]: This regex doesn't look right.

Comment (Spone) [reply lib/komponent/komponent_helper.rb]: `/^components/` ?

File: lib/komponent/komponent_helper.rb (hunk 1 above)
Comment (florentferry) [reply lib/komponent/komponent_helper.rb]: Seems this feature is already working on master branch.

**Pull request merged**